src/test/java/org/wecancodeit/reviews/controllers/ReviewControllerTest.java

```diff
@@ -14,7 +14,9 @@
 import static org.mockito.Mockito.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
 
-public class ReviewControllerTest {
+public class
+
+ReviewControllerTest {
```

Comment:
Please line this up

Reply:
Fixed in https://github.com/2020-Spring-Cohort/reviews-site-models/commit/4a8682409c75e91bd225174719d1faef4904f7a4